src/Microsoft.TestPlatform.Common/SettingsProvider/SettingsProviderExtensionManager.cs

```diff
@@ -115,7 +115,7 @@ public static SettingsProviderExtensionManager Create()
 
                         TestPluginManager.Instance
                             .GetSpecificTestExtensions<TestSettingsProviderPluginInformation, ISettingsProvider, ISettingsProviderCapabilities, TestSettingsProviderMetadata>(
-                                TestPlatformConstants.SettingsProviderEndsWithPattern,
+                                TestPlatformConstants.TestAdapterEndsWithPattern,
```

Comment:
>TestAdapterEndsWithPattern [](start = 54, length = 26)

Should we keep SetttingsProvider pattern as well, in case(extreme rare) for folks who might have used this pattern to define their ISettings

Reply:
Can we please add tests for this change if possible ?